cpp/include/cudf/strings/regex/flags.hpp

```diff
@@ -60,6 +61,17 @@ constexpr bool is_dotall(regex_flags const f)
   return (f & regex_flags::DOTALL) == regex_flags::DOTALL;
 }
 
+/**
+ * @brief Returns true if the given flags contain ASCII.
+ *
+ * @param f Regex flags to check
+ * @return true if `f` includes ASCII
+ */
+constexpr bool is_ascii(regex_flags const f)
```

Comment:
Also fine since it's copied, but couldn't all these functions accept the flags by reference?

Reply:
The flag is only a 32-bit value while a reference would require 64-bit stack variable. There is nothing gained by making it a reference.

Reply:
I see a type `regex_flags` and my brain assumes it's a big struct of some sort even though I've just seen the enum. Sorry for the noise.